www/src/pages/pl/t3-collection.md

```diff
@@ -22,6 +22,7 @@ Made a project using the T3 stack and want to share it? Add it to the list!
 | Ayanava's Guestbook                                               | [AyanavaKarmakar/Guestbook](https://github.com/AyanavaKarmakar/Guestbook)   | [guestbook.ayanavakarmakar.software](https://guestbook.ayanavakarmakar.software/) |
 | Slug - URL Shortener                                              | [pheralb/slug](https://github.com/pheralb/slug)                             | [slug.vercel.app](https://slug.vercel.app)                                        |
 | AI TTS Donations - FOSS AI Text To Speach service for Streamers.  | [mmattDonk/AI-TTS-Donations](https://github.com/mmattDonk/AI-TTS-Donations) | [staging.solrock.mmattDonk.com](https://staging.solrock.mmattDonk.com)            |
+| The Doom                                                          | [moltivie/slug](https://github.com/Moltivie/the-t3-stack)                   | [the-t3-stack.vercel.app](https://the-t3-stack.vercel.app)                        |
```

Comment:
we should probably make this an mdx component that can be imported into all languages so that we don't need to add a new entry to many places

Reply:
Yeah, right now I just try to be in sync with the English version. Will those projects' names be translated? If yes, that would also make it harder to extract the component (unless we create some projects array with i18n metadatas?)

Reply:
No i think we should stick with the projects original description?

Reply:
I don't mind it being in English. Some of them are more complex though (eg. Shoutify), but I think translating it is not that important for now